packages/icons/package.json

```diff
@@ -12,7 +12,7 @@
     "directory": "packages/icons"
   },
   "scripts": {
-    "build": "rm -rf ./lib && mkdir ./lib && cp -r ./scss/ ./lib/"
+    "build": "rimraf lib && mkdir lib && copyfiles --up 1 \"./scss/**/*.scss\" ./lib/"
```

Comment:
Can we use https://github.com/shelljs/shx instead of `rimraf` and `copyfiles` packages?

```
shx rm -rf ./lib && mkdir ./lib && shx cp -r ./scss/ ./lib/
```

Reply:
Sure... have made that change and tested on Win10. Will spin up a Ubuntu VM and check against Linux.

Reply:
And all good on a Mac.

Reply:
wonderful, thank you!